avaframe/in3Utils/fileHandlerUtils.py

```diff
@@ -326,23 +327,33 @@ def getDFADataPaths(avaDir, pathDict, suffix, comModule='', inputDir=''):
             path to avalanche directory
         pathDict: dict
             dictionary with paths to simulation results
+        cfg: configParser object
+            configuration for aimec, here varPar and ascendingOrder used
         suffix : str
             result parameter abbreviation (e.g. 'ppr')
         comModule : str
             optional - name of computational module (default is com1DFA)
-      """
+        inputDir: str
+            optional - path to a different input directory
+    """
 
     # Lead all infos on simulations
     if inputDir == '':
         inputDir = os.path.join(avaDir, 'Outputs', comModule, 'peakFiles')
         if os.path.isdir(inputDir) == False:
             log.error('Input directory does not exist - check anaMod')
 
-    data = makeSimDict(inputDir)
-    for m in range(len(data['files'])):
-        if data['resType'][m] == suffix:
-            pathDict[suffix].append(data['files'][m])
-            log.info('Added to pathDict: %s' % (data['files'][m]))
+    # fetch parameters that shall be used for ordering
+    varParList = cfg['varParList'].split('|')
+    # create dataFrame with ordered paths to simulation results
+    dataDF = cfgUtils.orderSimFiles(avaDir, inputDir, varParList, cfg['ascendingOrder'])
```

Comment:
maybe do this outside so that we do not do it again for each new peak value

Reply:
done